Cargo.toml.jinja

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "frequenz-{{project_name}}"
+description = "{{project_description}}"
+version = "0.1.0"
+edition = "2021"
+authors = ["Frequenz Energy-as-a-Service GmbH"]
+{% if open_source %}
+license = "MIT"
+{% else %}
+license = "proprietary"
+{% endif %}
```

Comment:
I think this is OK for now, but from our experience from the Python templates, I think it would be better to have a separate template for proprietary repos, as they tend to diverge quite a bit and make the template more complicated. Not sure if copier support template inheritance, and in particular inheriting from templates in other repos (cookiecutter support inheritance, but I think only from files in the same repo).

I think it might be worth pushing for this split sooner than later because, at least with the Python stuff, it seems to be more complicated to do the split as the repo grows.

Reply:
Multiple templates in one project: https://copier.readthedocs.io/en/stable/configuring/#applying-multiple-templates-to-the-same-subproject

Reply:
I would suggest splitting it into two separate templates now, and discussing internally first where to put the template for the proprietary crates.